pkg/services/ngalert/migration/service_test.go

```diff
@@ -279,7 +281,7 @@ func TestServiceRevert(t *testing.T) {
 		}
 	})
 
-	t.Run("ForceMigration story", func(t *testing.T) {
```

Comment:
If we keep the "force" flag, then I would recommend adding a new set of tests and keeping existing ones untouched.